packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -160,6 +162,7 @@ export function DevtoolsView(): React.ReactElement {
 			[DevtoolsFeatures.MessageType]: (untypedMessage) => {
 				const message = untypedMessage as DevtoolsFeatures.Message;
 				setSupportedFeatures(message.data.features);
+				clearTimeout(queryTimer.current); // Clear the timeout when found the supported features
```

Comment:
Do we need this here? If `supportedFeatures` changes, then the below Effect will re-run, and `clearTimeout` will be run while cleaning up the old Effect.

Reply:
If it turns out we do need this, then we need to add `queryTimer` to the Effect's dependency list.